locust/load-test/load-test.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+from locust import HttpUser, task, between
+import yaml, json, os
+
+class PuppetDbLoadTest(HttpUser):
+    def response_printer(self, opts, response):
+        if response.status_code == 0:
+            print(response.error)
+            exit(1)
+        elif response.status_code != 200:
+            print(
+            "Request: " + opts['name'] + " \n" +
+            "Method: " + opts['method'] + " \n" +
+            "Response status: " + str(response.status_code) + " \n" +
+            "Response body: " + response.text + " \n" +
+            "-------------------------------------------" + " \n")
```

Comment:
you could use something like this
`            print("Request: " + opts['name'],
                  "Method: " + opts['method'],
                  "Response status: " + str(response.status_code),
                  "Response body: " + response.text,
                  end='\n-------------------------------------------\n', sep='\n')`

Reply:
Updated, thanks!